### PR TITLE
Update exclusive-or.rst

### PR DESCRIPTION
**Pull request by nktrejo2020**

### CODE REVIEW

File: src/exclusive-or.rst [hunk 1]
```diff
@@ -321,52 +321,52 @@ some sequences that we know will occur very commonly, for example (the
...
-If we know more about the plaintext, we can make even better guesses.
-For example, if it's HTTP serving HTML, we would expect to see things
-like ``Content-Type``, ``<a>``, and so on.
-
-That only tells us which plaintext sequences are likely, giving us
-likely guesses. How do we tell if any of those guesses are correct? If
-our guess is correct, we know all the other plaintexts at that position
-as well, using the technique described earlier. We could simply look at
+We can make better guesses if more information is known about the plaintext.
+For example, if HTTP is serving HTML we expect to see
```
Comment (dg-latacora) [inline src/exclusive-or.rst:325]: maybe something like "HTML is served over HTTP"

File: src/exclusive-or.rst [hunk 2]
```diff
@@ -321,52 +321,52 @@ some sequences that we know will occur very commonly, for example (the
...
-methods by applying advanced, statistical models based on natural
-language analysis. This only demonstrates further just how broken
+the same key, it is clear that the process becomes simpler when more
+ciphertexts use the same key. Since all of the steps become more
+effective we get:
+
+-  More cribbing positions.
+-  More plaintext bytes revealed with each successful crib and
+   guess. This leads to more guessing options elsewhere.
+-  More ciphertexts available for any given position. This simplies guess
```
Comment (dg-latacora) [inline src/exclusive-or.rst:363]: typo

File: src/exclusive-or.rst [hunk 3]
```diff
@@ -375,28 +375,27 @@ Remaining problems
...
-secure symmetric encryption algorithms aren't the pain point of modern
-cryptosystems. Cryptographers have designed plenty of those, while
-practical key management remains one of the toughest challenges facing
-modern cryptography. One-time pads may solve a problem, but it's the
+that is not the case.
+
+One-time pads are rarely used for being horribly impractical.
+The key is at least as large as all information you would like transmitted,
+*put together*. Plus, the keys must be exchanged securely, ahead
+of time, with all people in your communication network. We like to
```
Comment (dg-latacora) [inline src/exclusive-or.rst:383]: "in your communication network" seems to change meaning for me by getting rid of strangers communicating

File: src/exclusive-or.rst (hunk 1 above)
Comment (lvh) [reply src/exclusive-or.rst:325]: Sounds good, go for it!

File: src/exclusive-or.rst (hunk 2 above)
Comment (lvh) [reply src/exclusive-or.rst:363]: Yep!

**Pull request closed without merging**